README.md

```diff
@@ -32,13 +32,22 @@ A modern zim fileserver which can handle multiple zim files by serving a beautif
  
 
 # Installation
-- Install go and compile it using `go build -o ZimWiki`
+- Install go and compile it using `go build`
 or
 - Download the latest release
 
+# Configuration
+### Example
+config.toml:
+```toml
+[Config]
+  LibraryPath = "./library"
+  Address = ":8080"
```

Comment:
Are those spaces added on purpose? Afaik is toml not indented with spaces at all

Reply:
Indentation seems to be allowed but not required.
Since you're making the point, I guess you prefer I take them off?

Reply:
Yes, I think toml looks cleaner this way.

Reply:
Okay, in that case, it's corrected.